include/tvm/tir/schedule/schedule.h

```diff
@@ -430,9 +430,11 @@ class ScheduleNode : public runtime::Object {
    * \param block_rv The block to be moved
    * \param loop_rv The loop where the block to be moved under
    * \param preserve_unit_loops Whether to keep the trivial loops whose extents are 1
+   * \param index The block index of the loop body subtree blocks
+   * -1, -2 means keep closed to or away from it's consumer
```

Comment:
I polished the doc a little bit. let me know if makes sense

```suggestion
   * \param index The block index of the loop body subtree blocks.
   * A nonnegative number means the index of a possible insertion point
   * -1 means inserted into the last possible insertion point
   * -2 means inserted into the first possible insertion point
```

Reply:
Agree. Thanks.